src/styles/_catalog.scss

```diff
@@ -1,10 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
If u follow my suggestion above, here u can create only 1 @media - repeat (auto-fill, 200px)

Reply:
And u don't need nth child, grid has done it all for u))

Reply:
agree, nice suggestion